From 8cfda450118f441458c7d3f32e22248e34334265 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Wed, 7 Aug 2019 12:25:04 +0100
Subject: [PATCH 1/1] Fix AVX2 build on non-AVX2 systems

The AVX2 build will try to run the AVX-built sln to make symbolic links.
Instead, use the system version which will always be available. This fixes
the AVX2 build on systems without AVX2 instructions.

Signed-off-by: Joey Riches <josephriches@gmail.com>
---
 Makerules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makerules b/Makerules
index 83bdd3a4..e5fe2bc0 100644
--- a/Makerules
+++ b/Makerules
@@ -1034,7 +1034,7 @@ endif
 
 ifeq (yes,$(build-shared))
 ifeq (no,$(cross-compiling))
-symbolic-link-prog := $(elf-objpfx)sln
+symbolic-link-prog := sln
 symbolic-link-list := $(elf-objpfx)symlink.list
 define make-shlib-link
 echo `$(..)scripts/rellns-sh -p $< $@` $@ >> $(symbolic-link-list)
-- 
2.22.0

